src/test/java/org/fundacionjala/coding/MarcoM/HighAndLowTest.java

```diff
@@ -0,0 +1,24 @@
+package org.fundacionjala.coding.MarcoM;
+
+import org.junit.Test;
+
+import static org.junit.Assert.assertEquals;
+
+/**
+ * Created by Administrator on 8/22/2017.
+ */
+public class HighAndLowTest {
```

Comment:
@AT-05/at-05 
please you need to ensure that the following scenarios pass.
please update your unit tests.

```java
public class HighAndLowTest {
    
    private HighAndLow highAndLow;

    @Before
    public void setUp() {
        highAndLow = new HighAndLow();
    }

    @Test
    public void testHighAndLowSome() {
        assertEquals("542 -214", highAndLow.highAndLow("4 5 29 54 4 0 -214 542 -64 1 -3 6 -6"));
    }

    @Test
    public void testHighAndLowPlusMinus() {
        assertEquals("1 -1", highAndLow.highAndLow("1 -1"));
    }

    @Test
    public void testHighAndLowPlusPlus() {
        assertEquals("1 1", highAndLow.highAndLow("1 1"));
    }

    @Test
    public void testHighAndLowMinusMinus() {
        assertEquals("-1 -1", highAndLow.highAndLow("-1 -1"));
    }

    @Test
    public void testHighAndLowPlusMinusZero() {
        assertEquals("1 -1", highAndLow.highAndLow("1 -1 0"));
    }

    @Test
    public void testHighAndLowPlusPlusZero() {
        assertEquals("1 0", highAndLow.highAndLow("1 1 0"));
    }

    @Test
    public void testHighAndLowMinusMinusZero() {
        assertEquals("0 -1", highAndLow.highAndLow("-1 -1 0"));
    }

    @Test
    public void testHighAndLowSingle() {
        assertEquals("42 42", highAndLow.highAndLow("42"));
    }
}

```